### PR TITLE
Fix alert about delay between best and finalized block

### PR DESCRIPTION
**Pull request by tomaka**
Right now there is a small fuck up on Polkadot, and the difference between the best and finalized block is of 50 blocks.

This should have triggered this alert, but didn't. That's because there is a typo in the alert, which this PR fixes.

If this PR gets merged, the alert will triggered, so I would suggest to not merge it until the finalization problem of Polkadot is fixed.

### CODE REVIEW

File: primitives/state-machine/src/proving_backend.rs [hunk 1]
```diff
@@ -249,7 +249,9 @@ impl<'a, S, H> Backend<H> for ProvingBackend<'a, S, H>
 	type TrieBackendStorage = S;
 
 	fn storage(&self, key: &[u8]) -> Result<Option<Vec<u8>>, Self::Error> {
-		self.0.storage(key)
+		let value = self.0.storage(key);
```
Comment (andresilva) [inline primitives/state-machine/src/proving_backend.rs:252]: Did you intend to add this?

**Review by andresilva — APPROVED** [review]

File: primitives/state-machine/src/proving_backend.rs (hunk 1 above)
Comment (tomaka) [reply primitives/state-machine/src/proving_backend.rs:252]: :facepalm: That's my debugging.

**Review by gabreal — APPROVED** [review]

**Comment by andresilva** [comment]
bot merge force

**Comment by andresilva** [comment]
Sorry I had not noticed that you didn't want to merge this yet. Good thing the bot didn't go through.

**Comment by tomaka** [comment]
Finalization issue is solved.

**Comment by tomaka** [comment]
bot merge

**Pull request merged**